src/api/Ethereum.ts

```diff
@@ -37,6 +38,14 @@ export type Tx = {
     }>;
     truncated: boolean;
   };
+  erc721TransferEvents?: {
+    list: Array<{
+      contract: string;
+      from: string;
+      to: string;
+      tokenId: BigNumber;
```

Comment:
tokenId better be a string 🤔

Reply:
From the real spec, it's supposed to be a UINT256, but what we are getting from the API is a number is think ? (In the transfer_events.lists the `count` is a number at least and since the ERC20 and ERC721 transfer event is the same, the backend is actually indexing NFT transfer events as ERC20 and putting the `tokenId` as `count`. So I guess it would be a number too ?) 

But nonetheless, we are casting those `Tx` directly from the `fetchAllTransactions` method which means that none of those non serialized types are wrong since we are not deserializing before hand.

Reply:
⚠️ Conversation is not outdate, `tokenId` has just been replaced by `token_id`, but the typing issue still remains.